app/models/user.rb

```diff
@@ -2,9 +2,70 @@ class User < ApplicationRecord
   has_many :posts, dependent: :destroy
   has_many :comments, dependent: :destroy
   has_many :likes, dependent: :destroy
+  has_many :friendships, foreign_key: "creator_id", dependent: :destroy
+  has_many :inverse_friendships, class_name: "Friendship", foreign_key: "recipient_id", dependent: :destroy
```

Comment:
- [ ] `**[OPTIONAL]**` We can easily create a relation to the confirmed requests `has_many :confirmed_friendships, -> { where status: true }, class_name: 'Friendship`. This will pave the way to find the friends for each user more easily. Please see my [comment]() below.